### ISSUE TITLE
On which port do I access Xenforo with the example compose file given?

### ISSUE THREAD

**Issue by wronda** [xfrocks/docker-xenforo#17]
Ah think I found it :) Was serving a default PHP.info page. In your example folder the docker-compose file doesn't include mysql. Is that correct?

**Comment by daohoangson** [comment]
Yes, [the example](https://github.com/xfrocks/docker-xenforo/blob/master/examples/docker-compose.yml) doesn't include MySQL or XF source (that would be illegal).

The `php_info()` will be available at a few different URLs:

- http://caddy.local.xfrocks.com caddy -> php-fpm
- https://caddy.local.xfrocks.com caddy -> php-fpm
- http://nginx.local.xfrocks.com nginx -> php-fpm
- http://apache.local.xfrocks.com php-apache

**Comment by xtrc** [comment]
And how would I connect to a database I imported. What do I set as host value in the config.php or init.php?

**Comment by daohoangson** [comment]
Just put in the `config.php` file as normal.

**Comment by xtrc** [comment]
Ok one more thing: https://world1.khoori.org/ doesn't work but https://world1.khoori.org/forum does. is the pdo_mysql part of the php-fpm image?

**Comment by daohoangson** [comment]
No, `pdo_mysql` extension is not installed because XF doesn't use it. The list of installed extension is available here https://github.com/xfrocks/docker-xenforo

**Comment by xtrc** [comment]
Got it :) One more thing. The nginx default configs are not copied into the nginx container. Something wrong with the entrypoint perhaps?

**Comment by daohoangson** [comment]
The nginx image generates its own configuration using this template:

- https://github.com/xfrocks/docker-xenforo/blob/master/nginx/templates/nginx.conf
- https://github.com/xfrocks/docker-xenforo/blob/master/nginx/templates/default.conf

It does not use the nginx default conf files FYI.
